modules/porous_flow/include/dirackernels/PorousFlowLineGeometry.h

```diff
@@ -69,6 +70,14 @@ class PorousFlowLineGeometry : public DiracKernel
   /// Reads a space-separated line of floats from ifs and puts in myvec
   bool parseNextLineReals(std::ifstream & ifs, std::vector<Real> & myvec);
 
+  ///@{
+  /// reporter input alternative (to the point file and line_base data)
+  const std::vector<Real> * _x_coord = nullptr;
+  const std::vector<Real> * _y_coord = nullptr;
+  const std::vector<Real> * _z_coord = nullptr;
+  const std::vector<Real> * _weight = nullptr;
```

Comment:
```suggestion
  const std::vector<Real> * const _x_coord;
  const std::vector<Real> * const _y_coord;
  const std::vector<Real> * const _z_coord;
  const std::vector<Real> * const _weight;
```

Remove the default value; makes for unclear code because you're setting it in the ctor anyway.

Add const (const pointer to a const vec) because you're setting this in the ctor (both the vector and the pointer can then not be changed)

Reply:
Thanks Logan for explaining the extra const.  I keep forgetting.